cmd/notary/integration_test.go

```diff
@@ -81,6 +81,98 @@ func setupServer() *httptest.Server {
 	return httptest.NewServer(setupServerHandler(storage.NewMemStorage()))
 }
 
+// Initializes a repo with existing key
+func TestInitWithRootKey(t *testing.T) {
+	// -- setup --
+	setUp(t)
+
+	tempDir := tempDirWithConfig(t, "{}")
+	defer os.RemoveAll(tempDir)
+
+	server := setupServer()
+	defer server.Close()
+
+	tempFile, err := ioutil.TempFile("", "targetfile")
+	require.NoError(t, err)
+	tempFile.Close()
+	defer os.Remove(tempFile.Name())
+
+	// -- tests --
+
+	// create encrypted root key
+	privKey, err := trustmanager.GenerateECDSAKey(rand.Reader)
+	require.NoError(t, err)
+	encryptedPEMPrivKey, err := trustmanager.EncryptPrivateKey(privKey, data.CanonicalRootRole, testPassphrase)
+	require.NoError(t, err)
+	encryptedPEMKeyFilename := filepath.Join(tempDir, "encrypted_key.key")
+	err = ioutil.WriteFile(encryptedPEMKeyFilename, encryptedPEMPrivKey, 0644)
+	require.NoError(t, err)
+
+	// init repo
+	_, err = runCommand(t, tempDir, "-s", server.URL, "init", "gun", "--rootkey", encryptedPEMKeyFilename)
+	require.NoError(t, err)
+
+	// publish repo
+	_, err = runCommand(t, tempDir, "-s", server.URL, "publish", "gun")
+	require.NoError(t, err)
+
+	// check that the root key used for init is the one listed as root key
+	output, err := runCommand(t, tempDir, "key", "list")
+	require.NoError(t, err)
+	t.Logf(output)
+	require.True(t, strings.Contains(output, data.PublicKeyFromPrivate(privKey).ID()))
+
+	// check error if file doesn't exist
+	_, err = runCommand(t, tempDir, "-s", server.URL, "init", "gun2", "--rootkey", "bad_file")
+	require.Error(t, err, "Init with nonexistent key file should error")
+
+	// check error if file is invalid format
+	badKeyFilename := filepath.Join(tempDir, "bad_key.key")
+	nonPEMKey := []byte("thisisnotapemkey")
+	err = ioutil.WriteFile(badKeyFilename, nonPEMKey, 0644)
+	require.NoError(t, err)
+
+	_, err = runCommand(t, tempDir, "-s", server.URL, "init", "gun2", "--rootKey", badKeyFilename)
```

Comment:
I think the flag for these next few test cases should have `--rootkey` as opposed to `--rootKey`?